scripts/test_next_version.py

```
@@ -0,0 +1,200 @@
+        if "/compare/" in url:
+            if url.endswith("&page=1"):
+                return {"commits": [{"sha": "aaa"}, {"sha": "bbb"}], "total_commits": 5}
+            return {"commits": [], "total_commits": 5}
+        return []
+    monkeypatch.setattr(nv, "_github_get", fake_get)
+    nv.fetch_pull_requests("aegro/example", "from_sha", "to_sha", "token")
+    assert "compare truncated at 2 of 5 commits" in capsys.readouterr().err
+
+
+def test_fetch_pull_requests_is_quiet_when_compare_is_complete(monkeypatch, capsys):
+    def fake_get(url, token):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `fake_get`

(ANN202)